docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -9,281 +9,350 @@ priority: 2
 
-```scala
-> val s = sc.textFile("alluxio://zk@zkHost1:2181;zkHost2:2181;zkHost3:2181/LICENSE")
-> val double = s.map(line => line + line)
-> double.saveAsTextFile("alluxio://zk@zkHost1:2181;zkHost2:2181;zkHost3:2181/LICENSE2")
+```bash
+$ ./bin/alluxio-stop.sh masters # stops all masters in conf/masters
+$ ./bin/alluxio-stop.sh workers # stops all workers in conf/workers
 ```
 
-Alluxio will help you set Zookeeper properties and find Alluxio leader master.
+If you do not want to use `ssh` to login to all the nodes and stop all the processes, you can run
+commands on each node individually to stop each component. For any node, you can stop a master or worker with:
 
-If both Zookeeper configuration is set and Alluxio URI is provided, the connection details 
-in the URI will be used for finding the Alluxio leader master.
+```bash
+$ ./bin/alluxio-stop.sh master # stops the local master
+$ ./bin/alluxio-stop.sh worker # stops the local worker
+```
+
+Starting Alluxio is similar. If `conf/workers` and `conf/masters` are both populated, you can start
+the cluster with:
 
-#### Automatic Fail Over
+```bash
+$ ./bin/alluxio-start.sh all SudoMount
+```
 
-To test automatic fail over, ssh into current Alluxio master leader, and find process ID of
-the `AlluxioMaster` process with:
+You can start just the masters and just the workers with the following commands:
 
 ```bash
-$ jps | grep AlluxioMaster
+$ ./bin/alluxio-start.sh masters           # starts all masters in conf/masters
+$ ./bin/alluxio-start.sh workers SudoMount # starts all workers in conf/workers
 ```
 
-Then kill the leader with:
+If you do not want to use `ssh` to login to all the nodes and start all the processes, you can run
+commands on each node individually to start each component. For any node, you can start a master or worker with:
 
 ```bash
-$ kill -9 <leader pid found via the above command>
+$ ./bin/alluxio-start.sh master           # starts the local master
+$ ./bin/alluxio-start.sh worker SudoMount # starts the local worker
 ```
 
-Then you can see the leader with the following command:
+## Format the Journal
+
+On any master node, format the Alluxio journal with the following command:
 
 ```bash
-$ ./bin/alluxio fs leader
+$ ./bin/alluxio format
 ```
 
-The output of the command should show the new leader. You may need to wait for a moment for the
-new leader to be elected.
+> Formatting the journal will delete all metadata from Alluxio. However, the data in under storage will be untouched.
+
+## Add/Remove Workers
+
+Adding a worker to an Alluxio cluster is as simple as starting a new Alluxio worker process, with the
+appropriate configuration. In most cases, the new worker's configuration should be the same as all
+the other workers' configuration. Once the worker is started, it will register itself with the Alluxio
+master, and become part of the Alluxio cluster.
+
+Removing a worker is as simple as stopping the worker process. Once the worker is stopped, the master will
```

Comment:
we should mention that there is a delay/timeout, otherwise users might expect the worker to immediately show up as lost after they stop it

Reply:
Done